CONTRIBUTING.md

```diff
@@ -108,6 +108,8 @@ This library is built with Gradle.
   things up during development.
 * Make sure to pass the extra parameter `-Pkotlin.dataframe.debug=true` to enable debug mode. This flag will
   make sure some extra checks are run, which are important but too heavy for production.
+* Parameter `-PskipKodex` allows you to skip [kdoc processing](KDOC_PREPROCESSING.md), making local publishing faster
```

Comment:
*The parameter

Reply:
I miss a small part here or in the other place maybe that you generally don't want to use this setting as it produces a library with "broken" KDocs; it's only for faster development if you don't care about the KDocs